tailcfg/derpmap.go

```diff
@@ -92,6 +92,9 @@ type DERPRegion struct {
 	// "San Francisco", "Singapore", "Frankfurt", etc.
 	RegionName string
 
+	// Location is an optional tailcfg.Location which represents geographical location about a DERP region.
```

Comment:
I feel like this says the same thing in a few different ways, we can make it more concise.

```suggestion
	// Location optionally describes where the DERP region is located.
```